.eslintignore

```diff
@@ -9,3 +9,4 @@ website/
 scripts
 packages/docusaurus-1.x/lib/core/metadata.js
 packages/docusaurus-1.x/lib/core/MetadataBlog.js
+packages/docusaurus-1.x/lib/core/__tests__/split-tab.test.js
```

Comment:
Added because the lint fixer keeps removing `@jest-environment jsdom` from my test file. I'm sure there's a workaround so this is just temporary.